src/modm/platform/core/rp/module.md

```diff
@@ -0,0 +1,16 @@
+# RP2040 core module
+
+This module provide RP2040-specific startup code and linkerscripts.
+
+
+## Startup
+
+The `__modm_initialize_platform()` callback is implemented here to reset
+hardware to known initial state.
+
+
+## Linkerscript
+
+Currently only one basic linkerscript is supported.
```

Comment:
Once we figure out how to do dual-core nicely we can update this

Reply:
My written English is not good enough for an easy to understand description. Can you help with this?

Reply:
Sure, no problem, I'll add a description once everything in this PR is settled.